yarn.lock

```diff
@@ -4312,15 +4312,6 @@
     resolve-from "^5.0.0"
     store2 "^2.7.1"
 
-"@superset-ui/chart-controls@0.17.22":
```

Comment:
Why do we need to change the lock file here?

Reply:
done. rebase on master